linux_os/guide/services/ftp/ftp_configure_vsftpd/ftp_configure_firewall.rule

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+title: 'Configure Firewalls to Protect the FTP Server'
+
+description: |-
+    By default, <tt>iptables</tt>
```

Comment:
Should probably add this to the existing jinja template to differentiate between `iptables` and `firewalld`